js/index.js

```diff
@@ -0,0 +1,21 @@
+window.addEventListener('DOMContentLoaded', () => {
+    const btn = document.querySelector('#menu_btn');
+    console.log(btn);
+    const menu = document.querySelector('#mobile-nav');
+    console.log(menu);
+    const close = document.querySelector('#mobile-nav span');
+    console.log(close);
```

Comment:
- [ ] As the previous reviewer mentioned, please remove `console.log( )`  from your code because It is killing the performance and you can take a look in [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)